src/downloadmanager.cpp

```diff
@@ -927,6 +927,31 @@ void DownloadManager::visitOnNexus(int index)
   }
 }
 
+void DownloadManager::openFile(int index)
+{
+  if ((index < 0) || (index >= m_ActiveDownloads.size())) {
+    reportError(tr("VisitNexus: invalid download index %1").arg(index));
+    return;
+  }
+  QString params = "/select,\"";

```

Comment:
select is only for explorer to actually select a file in the explorer window, so this can be removed

Reply:
Gone.